### ISSUE TITLE
Error occurred while scanning

### ISSUE THREAD

**Issue by dhuff44** [RandomEngy/VidCoder#37]
I keep getting Error occurred while scanning.  I have tried different disk, and different disk drives.  Any thoughts?

**Comment by RandomEngy** [comment]
Hard to say for sure without any additional information. Are you trying to scan a copy-protected DVD or Blu-Ray?

**Comment by dhuff44** [comment]
Yes, I am trying to back up my p90x3 disk before they get destroyed, I have
kids.  I am also using Passkey 8.

**Comment by RandomEngy** [comment]
Hmm, so that should remove the encryption. Do you get the same issue with HandBrake? Want to find out if this is a VidCoder specific error.

**Comment by dhuff44** [comment]
I have not tried Handbrake, never really understood it.

**Comment by RandomEngy** [comment]
Okay if you can get me the source file you're trying to scan I can take a look at it.

**Comment by osexpert** [comment]
Would like option to disable preview scanning completely, if often fails or is slow for various reasons:
- scanning a very short video clip with approx less than 20 frames always fails
- scanning file where video is (much) shorter than audio fails
In both cases the files can be converter without problem in handbrake, but in VidCoder the preview scanning crashes and makes it impossible to queue the file.

**Comment by RandomEngy** [comment]
@osexpert Can you send me some of the short clips that fail so I can test with them?

**Comment by osexpert** [comment]
The very short clips fail in HB too.
The other (short video with much longer audio track) I can't reproduce now.
